AUTHORS.txt

```diff
@@ -10,5 +10,9 @@ Ben Feinstein
 Neal Reynolds
 Christoph Buchner
 Julian Dlugosch
+<<<<<<< HEAD
```

Comment:
Please fix these artifacts from the git merge, so that this file only contains a list of author names. Also please add yourself to `pymeasure/docs/about/authors.rst`.